src/index.html

```diff
@@ -6,6 +6,17 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>React and Webpack4</title>
+  <style>
```

Comment:
I think we need a better solution here, i dont like styles defined in index.html

Reply:
Let's discuss later.